thirdparty/traccc/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ set(TRACCC_SETUP_ALGEBRA_PLUGINS
 )
 set(TRACCC_SETUP_COVFIE OFF CACHE BOOL "Do not set up Covfie as part of Traccc")
 set(TRACCC_SETUP_DFELIBS
-    OFF
+    ON
```

Comment:
Why is that change? Otherwise good from my side.

Reply:
So the dfelibs repository is now deprecared (@paulgessinger) and we've moved the ACTS and Detray repositories away from it. I tried to also migrate the traccc repository but this did not fly (see https://github.com/acts-project/traccc/pull/893) so for now we need traccc to explicitly download its own copy of dfelibs.